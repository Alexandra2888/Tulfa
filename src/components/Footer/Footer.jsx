import Description from "../../atoms/Description/Description";

const Footer = () => {
  return (
    <section className="py-24">
      <Description
        text="Lorem ipsum dolor sit amet consectetur. Augue elementum morbi in ac. Leo
        eu elit nibh nunc vitae eget massa sed sed. Sit sed aliquam sit nulla
        eget."
      />
    </section>
  );
}

export default Footer
