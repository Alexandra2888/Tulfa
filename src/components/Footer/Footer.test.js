import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import Footer from "./Footer";

// Mock the Description component in a simpler way
jest.mock("../../atoms/Description/Description", () => {
  return function MockDescription({ text }) {
    return <div data-testid="description">{text}</div>;
  };
});

describe("Footer", () => {
  it("renders without crashing", () => {
    render(<Footer />);
    expect(screen.getByTestId("description")).toBeInTheDocument();
  });

  it("renders with correct text content", () => {
    render(<Footer />);
    expect(screen.getByTestId("description")).toHaveTextContent(
      "Lorem ipsum dolor sit amet consectetur. Augue elementum morbi in ac. Leo eu elit nibh nunc vitae eget massa sed sed. Sit sed aliquam sit nulla eget."
    );
  });
});
