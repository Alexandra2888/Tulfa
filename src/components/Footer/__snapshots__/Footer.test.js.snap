// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<div>
  <section
    class="py-24 md:py-18 xl:py-18"
  >
    <div
      data-testid="mock-description"
    >
      Lorem ipsum dolor sit amet consectetur. Augue elementum morbi in ac. Leo
 eu elit nibh nunc vitae eget massa sed sed. Sit sed aliquam sit nulla
 eget.
    </div>
  </section>
</div>
`;
